Multisig/UI/Settings/OwnerKeyManagement/OwnerKeysListViewController/OwnerKeysListTableViewCell.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  OwnerKeysListTableViewCell.swift
+//  Multisig
+//
+//  Created by Moaaz on 3/9/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+protocol OwnerKeysListTableViewCellDelegate {
+    func ownerKeysListTableViewDidEdit(cell: OwnerKeysListTableViewCell)
+    func ownerKeysListTableViewCellDidRemove(cell: OwnerKeysListTableViewCell)
+}
+class OwnerKeysListTableViewCell: UITableViewCell {
+    @IBOutlet private weak var addressInfoView: AddressInfoView!
+    var delegate: OwnerKeysListTableViewCellDelegate?
```

Comment:
delegate must be weak, otherwise we'll have a retain cycle.

To enable this, you need to mark protocol as class:

```
protocol OwnerKeysListTableViewCellDelegate: class {
```